docs/release/3.4/release-3-4-0-2.md

```diff
@@ -0,0 +1,146 @@
+OSG Software Stack -- Data Release -- 3.4.0-2 and 3.3.25-2
+==========================================================
+
+**Release Date**: 2017-06-15
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   CA Certificates based on [IGTF 1.83](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    -   Added new trust anchor for accredited KISTI CA v3 (KR)
+    -   Removed obsolete GEANT TCS G1 and G2 (old Comodo-backed) trust anchors:
+        -   UTN-USERFirst-Hardware
+        -   TERENA-eScience-SSL-CA
+        -   AAACertificateServices
+        -   UTNAAAClient
+        -   TERENAeSciencePersonalCA
+        -   UTN-USERTrust-RSA-CA
+        -   TERENA-eScience-SSL-CA-2
+        -   TERENAeSciencePersonalCA2 (EU)
+-   [VO Package v74](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-74)
+    -   Fix the edg-mkgridmap entries for project8 and miniclean
+    -   Add new VOMS entry for CIGI
+    -   Add LIGO entry to GUMS template
+    -   Fix vo-client ATLAS mappings
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.4.0-2%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Detailed changes are below. All of the documentation can be found in the [Release3](https://twiki.grid.iu.edu/bin/view/Documentation/Release3/) area of the TWiki.
+
+Updating to the new release
+---------------------------
+
+### Update Repositories
+
+To update to this series, you need [install the current OSG repositories](../../common/yum#install-osg-repositories).
+
+### Update Software
+
+Once the repositories are installed, you can update to this new release with:
+
+``` console
+[root@client ~] $ yum update
+```
+
+<span class="twiki-macro NOTE"></span> Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
```

Comment:
Scrub out all the `twiki-macro` spans.  Above is an example of something that `pandoc` missed, but there's typically one or two per converted document.